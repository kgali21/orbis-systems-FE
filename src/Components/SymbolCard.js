import React from 'react';
import Proptypes from 'prop-types';

const Symbol = ({ symbol, tweet, tweetCount}) => (
    <div>
        <h1>{symbol}</h1>
        <p>{tweet}</p>
        <p>{tweetCount}</p>
    </div>
);

Symbol.propTypes = {
    symbol: Proptypes.string.isRequired,
    tweet: Proptypes.string.isRequired,
    tweetCount: Proptypes.string.isRequired,
};

export default Symbol
