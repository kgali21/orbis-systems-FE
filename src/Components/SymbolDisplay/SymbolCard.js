import React from 'react';
import PropTypes from 'prop-types';
import styles from './SymbolCard.css';

const Symbol = ({ symbol, tweet, tweetCount}) => (
    <div className={styles.Card}>
        <h1>{symbol}</h1>
        <p>{tweet}</p>
        <p>{tweetCount}</p>
    </div>
);

Symbol.propTypes = {
    symbol: PropTypes.string.isRequired,
    tweet: PropTypes.string.isRequired,
    tweetCount: PropTypes.number.isRequired
};

export default Symbol
