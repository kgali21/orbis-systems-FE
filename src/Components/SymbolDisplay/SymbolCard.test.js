import React from 'react';
import { shallow } from 'enzyme';
import SymbolCard from './SymbolCard';

describe('Messages Component', () => {
    it('renders Messages Component', () => {
        const wrapper = shallow(<SymbolCard symbol={'AAPL'} tweet={{}} tweetCount={12}/>);
        expect(wrapper).toMatchSnapshot();
     });
    }
);