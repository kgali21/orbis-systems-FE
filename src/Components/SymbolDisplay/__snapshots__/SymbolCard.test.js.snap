// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages Component renders Messages Component 1`] = `ShallowWrapper {}`;
